README.md

```diff
@@ -907,7 +911,7 @@ Returns the index of the last occurrence of value in the array, or -1 if value i
 **[⬆ back to top](#quick-links)**
 
 ### _.reverse
-:heavy_exclamation_mark:`Lodash only`
```

Comment:
I think this means underscore doesn't have this method (doesn't imply cant implement is natively).

Reply:
@stevemao Oh, well, if that's the case, then I'll put it back then. Sorry, I misinterpreted what it could mean, especially as it's apparently likely not the first time it was misinterpreted (e.g. Issue #110 mentioned in the PR).

I'll put that back. Thanks for the clarification.

Reply:
@stevemao Perhaps instead of `Lodash only`,it could be `Not in Underscore.js`?

Of course, obviously I'm just a random contributor and not one of the maintainers, so I don't want to sound like I'm lecturing. I'm just making a suggestion for a possible response to  Issue #110.

Reply:
Of course, you could just mark the Issue as closed and chalk it up to a misunderstanding.

Reply:
@cjtaylor1990 Thanks for the suggestion. Please go ahead with whatever you think is the best :)

Reply:
I just sent you an invite to the team :)

Reply:
@stevemao I have changed it to `Not in Underscore.js` as I find that it is still short and sweet, but it gets the point across a bit better given that you are talking about three things (a native implementation, Lodash, and Underscore).

Also, thank you for inviting me to the You-Dont-Need team. I started helping since I found some of the Issues to be fun to solve, and I've since came to really enjoy contributing where I can.